Common/UnitEnumValues.g.json

```diff
@@ -1334,7 +1334,9 @@
     "GramPerKiloNewtonSecond": 1,
     "KilogramPerKilogramForceHour": 2,
     "KilogramPerKiloNewtonSecond": 3,
-    "PoundMassPerPoundForceHour": 4
+    "PoundMassPerPoundForceHour": 4,
+    "GramPerKilonewtonSecond": 11,
+    "KilogramPerKilonewtonSecond": 6
```

Comment:
Same here: 
> This maybe a little hack-y, but I think that if we changed these by hand (to the old values) - I think they would stick.

Reply:
Yup, I suggest we do the same here, match the old values.

Reply:
fixed in https://github.com/angularsen/UnitsNet/pull/1491/commits/730eab19ddb691296c3c6cadd1278ac24f87ed89